source/monitoring.txt

```diff
@@ -180,6 +213,28 @@ To learn how to enable JMX for a containerized {+kc+}
 deployment, see
 `Kafka Monitoring and Metrics Using JMX with Docker <https://docs.confluent.io/platform/current/installation/docker/operations/monitoring.html>`__.
 
+.. _kafka-monitoring-types-of-metrics:
+
+Types of Metrics
+~~~~~~~~~~~~~~~~
+
+The connector provides metrics related to the following
+types of quantities:
+
+- The number of times an event has occurred in total for a connector task
+- The value related to the most recent occurrence of an event
+
+For some use cases, you must perform extra computations with the
+metrics the connector provides. For example, you must compute the
+following values from provided metrics:
+
+- The rate of change of a metric
+- The value of a metric over a span of time
+- How does one metric compare to another metric    
```

Comment:
Nit: "How one metric compares to another metric" ? (Something seems different about the phrasing of this 3rd item compared to the prior two)